source/Applications/Basic/FileFormats/ReadIterateZDF/Properties/AssemblyInfo.cs

```diff
@@ -28,9 +29,7 @@
 //      Minor Version
 //      Build Number
 //      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+
```

Comment:
My  suggestion was to just keep the existing comment (in all the AssemblyInfo.cs)
```
// You can specify all the values or you can default the Build and Revision Numbers
// by using the '*' as shown below:
// [assembly: AssemblyVersion("1.0.*")]
```

Now you just removed it. Look at git show, no reason to change any comments. And when reading the comment, It seems you need to add asteriks like so:

[assembly: AssemblyVersion("1.0.*")]

Same for AssemblyFileVersion and AssemblyInformationalVersion.

Reply:
Or just do "1.0.0.0" and don't use asteriks (probably the best)

Reply:
Done, 1.0.0.0 with no comments about default values